spring-cloud-starter-stream-processor-tensorflow/src/main/java/org/springframework/cloud/stream/app/tensorflow/processor/TensorFlowService.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.tensorflow.processor;
+
+import static org.apache.commons.io.IOUtils.buffer;
+import static org.apache.commons.io.IOUtils.toByteArray;
```

Comment:
No `static` imports in the `main` classes.
Anyway even tests have a restricted list of allowed static imports.

Also consider to keep as much Spring API as possible:

`StreamUtils.copyToByteArray()` does the trick for you.